perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -244,6 +269,19 @@ def CleanupBucket(storage_schema):
     objects_to_cleanup = _ListObjects(storage_schema, FLAGS.bucket, prefix=None)
 
 
+def GenerateWritePayload(size):
+  payload_bytes = bytearray(size)
+  # don't use a for...range(100M), range(100M) will create a list of 100mm items
+  # with each item being the default object size of 30 bytes or so, it will lead
+  # to out of memory error.
+  for i in xrange(size):
+    payload_bytes[i] = ord(random.choice(string.letters))
+
+  payload_string = payload_bytes.decode('ascii')
+
+  return payload_bytes, payload_string
```

Comment:
I will plan to submit that PR first and then rebase this on top of that, so the problem will go away before I submit this.